.github/ISSUE_TEMPLATE/releases/release_checklist.md

```diff
@@ -15,13 +15,14 @@
      - This is useful to make sure context is available to review later. It may end up blank if the release goes very smoothly.
      - There are other notes on servicing in this OneNote. It may be useful to review if something goes wrong to see if it's been fixed before.
      - File issues appropriately as you encounter problems, and link to them from the notes. Provide info in the issue rather than in the notes.
-1. - [ ] Retrieve dotnet/installer commit sha from internal communications for the release.
+1. - [ ] Retrieve the final run of the [Stage-DotNet](https://dev.azure.com/dnceng/internal/_build?definitionId=792&_a=summary) pipeline from internal release communications.
+1. - [ ] Run the [source-build-pre-release](https://dev.azure.com/dnceng/internal/_build?definitionId=1188) pipeline. When staging the pipeline run, click "Resources" and provide select the final run of Stage-DotNet mentioned above.
+     - [ ] When the pipeline finishes, look in the logs for the dotnet/installer commit sha that represents the final release.
+     - This pipeline automatically uploads the dotnet source tarball to dotnetclimsrc
 1. - [ ] Ensure the official installer and tarball builds have completed for the release's commit sha.
```

Comment:
Shouldn't this step be run prior to uploading the source tarball?

Reply:
I guess this should be another goal of the automation, to check that the installer and tarball builds were successful. I left it below the pre-release pipeline because the pipeline gives you the commit sha, and prevents you from having to look through the manifest.json yourself. It's not the end of the world to check it manually beforehand just for now though.

Reply:
Agreed we should be checking the success state.

Reply:
I added a check that the tarball build succeeded before continuing. If the build is red, then we fail the pipeline and require the pipeline run IDs to be manually provided.